src/main/java/com/github/may2beez/mayobees/config/MayOBeesConfig.java

```diff
@@ -860,6 +859,43 @@ public static long getRandomizedForagingMacroRotationSpeed() {
     )
     public static boolean antiAfkWhileFishing = false;
     //</editor-fold>
+
+    //<editor-fold desc="Enchanting">
+    @Switch(
+            name = "AutoExperiments",
+            description = "Automate Experimentation Table Mini Games",
+            category = "Skills",
+            subcategory = "Enchanting"
+    )
+    public static boolean autoExperiments = false;
+
+    @Slider(
+            name = "Click Delay",
+            description = "Delay between clicks during the Mini Games",
+            category = "Skills",
+            subcategory = "Enchanting",
+            min = 0, max = 1000
+    )
+    public static int autoExperimentsClickDelay = 400;
+
+    @Slider(
+            name = "Click Delay Randomization",
+            description = "Random Delay between clicks during the Mini Games",
+            category = "Skills",
+            subcategory = "Enchanting",
+            min = 0, max = 300
+    )
+    public static int autoExperimentsClickDelayRandomization = 150;
+
+    @Slider(
+            name = "Max Rounds",
+            description = "Maximum times to complete the mini games",
+            category = "Skills",
+            subcategory = "Enchanting",
+            min = 1, max = 40
+    )
+    public static int autoExperimentsMaxRounds = 9;
```

Comment:
Change this to metaphysical serum count. As the first mini game needs 12 without serum and second game needs 9 without serum. But first game only needs 12 at level 40, under it need 9, so check for that.